### PR TITLE
docs: created design guidelines for button group

### PR DESCRIPTION
**Pull request by orion-cengage**

### CODE REVIEW

**Review by ccedrone — APPROVED** [review]

File: website/react-magma-docs/src/pages/design/button-group.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      </figure>
+    </div>
+    <div class="col col-2">
+    </div>
+  </div>
+</div>
+
+### No Space
+
+You can easily remove the space between buttons to create a single unit. When you join buttons like this together, there typically isn't a hierarchy within their relationship, so it is acceptable to use the same button style.
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/design/button-group.mdx:59]: Maybe it's obvious, but should we mention that no space is only for horizontal?

File: website/react-magma-docs/src/pages/design/button-group.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
+---
+pageTitle: Button Group Design Guidelines
+title: Button Group
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/design/button-group.mdx:3]: Under API, we use `ButtonGroup`. Do we care that they are different?

File: website/react-magma-docs/src/pages/design/button-group.mdx (hunk 1 above)
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/button-group.mdx]: I have updated the copy with your feedback.

File: website/react-magma-docs/src/pages/design/button-group.mdx (hunk 2 above)
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/button-group.mdx]: Fixed

**Review by silvalaura — APPROVED** [review]

**Comment by Uditsingh67** [comment]
Verified design guidelines for button group.
**See Screen-cast :** http://somup.com/c3iF1KuYvB
RM Link : https://next--upbeat-sinoussi-f675aa.netlify.app/design/button-group/
Thanks!

**Pull request merged**